src/includes/configuration/config-intro/rust.mdx

```diff
@@ -1,7 +1,8 @@
 Options are passed to the `init()` function as tuple where the first argument is the DSN and the second the options:
 
 ```rust
-sentry::init(("___PUBLIC_DSN___", sentry::ClientOptions {
+let _guard = sentry::init(("___PUBLIC_DSN___", sentry::ClientOptions {
+    release: Some("my-project-name@2.3.12".into()),
```

Comment:
NOTE to self: use/recommend the provided macro which does this automatically

Reply:
Actually, good point. I keep forgetting about that one since it's not mentioned on docs 🤣 . Let's update this right away.